stable/swagger/README.md

```diff
@@ -0,0 +1,128 @@
+| `service.type`                           | Kubernetes service type                     | `TCP`                           |
+| `service.protocol`                       | Kubernetes service protocol                 | `NodePort`                      |
+| `service.externalPort`                   | Kubernetes external service port            | `80`                            |
+| `service.internalPort`                   | Kubernetes internal service port            | `8080`                          |
+| `ingress.enabled`                        | Enable ingress controller resource          | `false`                         |
+| `ingress.annotations`                    | Ingress annotations                         | `{}`                            |
+| `ingress.hostname`                       | URL to address Swagger                      | `nil`                           |
+| `ingress.tls`                            | Ingress TLS configuration                   | `[]`                            |
+| `ingress.basicAuth.enabled`              | Enable basic auth                           | `false`                         |
+| `ingress.basicAuth.secret`               | htpasswd username:password hash             | `YWRtaW46JGFwcjEkUm4zVHBwNDUkb2VOd0JoeWtWWHh0bUNmNnJ1Y2VaMAo=` which is the equivalent of `admin:test`, use `htpasswd -n <username>` to generate your own  |
+| `ingress.basicAuth.basicAuthAnnotations` | Ingress annotations specific to basic auth  | `{  nginx.ingress.kubernetes.io/auth-type: basic,    nginx.ingress.kubernetes.io/auth-realm: "Authentication Required"}`   |
+| `ingress.oauth.enabled`                  | Enable oauth auth                           | `false`                         |
+| `ingress.oauth.oauthAnnotations`         | Ingress annotations specific to oauth       | `{  kubernetes.io/ingress.class: nginx,   nginx.ingress.kubernetes.io/auth-signin: https://$host/oauth2/start,   nginx.ingress.kubernetes.io/auth-url: https://$host/oauth2/auth}`     |
+| `resources`                              | pod resource requests & limits              | `{}`                            |
+| `nodeSelector`                           | node labels for pod assignment              | `{}`                            |
+| `tolerations`                            | node taints to tolerate                     | `[]`                            |
+| `affinity`                               | node/pod affinities                         | `{}`                            |
+
+
+**Https and Nginx**
+
+An example configuration that uses `nginx-ingress` and `cert-manager` to provide a UI with https enabled. For more information on the dependecy configurations, please review the respective charts.
+```yaml
+ingress:
+  enabled: true
+  hostname: test.example.com
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: 'true'
+  tls:
+  - hosts:
```

Comment:
Please use a consistent list indentation style throughout the chart, either zero or two spaces.